lms/services/lti_grading/interface.py

```diff
@@ -38,6 +38,15 @@ def __init__(self, line_item_url: str, line_item_container_url: str):
         self.line_item_url = line_item_url
         self.line_item_container_url = line_item_container_url
 
+    def get_score_maximum(self, resource_link_id):  # pylint:disable=unused-argument
+        """
+        Read the grading configuration of an assignment.
+
+        In LTI nomenclature this is reading the line item container.
+        :param resource_link_id: ID of the assignment on the LMS.
+        """
+        return None
```

Comment:
In LTI1.1 we won't read this value for now.

Reply:
Is there even a way to get this in LTI 1.1?

Reply:
I don't think there is:

http://www.imsglobal.org/specs/ltiomv1p0/specification